src/main/resources/assets/minecolonies/lang/en_us.json

```diff
@@ -1053,61 +1050,61 @@
 
-  "item.minecolonies.barbarianegg": "Barbarian Egg",
-  "item.minecolonies.barbarcheregg": "Barbarian Archer Egg",
-  "item.minecolonies.barbchiefegg": "Barbarian Chief Egg",
-  "item.minecolonies.pirateegg": "Pirate Egg",
-  "item.minecolonies.piratearcheregg": "Pirate Archer Egg",
-  "item.minecolonies.piratecaptainegg": "Pirate Captain Egg",
-  "item.minecolonies.mercegg": "Mercenary Egg",
+  "item.minecolonies.barbarianegg": "Barbarian Spawn Egg",
+  "item.minecolonies.barbarcheregg": "Archer Barbarian Spawn Egg",
+  "item.minecolonies.barbchiefegg": "Chief Barbarian Spawn Egg",
+  "item.minecolonies.pirateegg": "Pirate Spawn Egg",
+  "item.minecolonies.piratearcheregg": "Archer Pirate Spawn Egg",
+  "item.minecolonies.piratecaptainegg": "Pirate Captain Spawn Egg",
+  "item.minecolonies.mercegg": "Mercenary Spawn Egg",
 
-  "com.minecolonies.coremod.pirates.sailing.away": "A pirate ship %s of %s just sailed away to search another victim.",
-  "com.minecolonies.coremod.cook.serve.player": "%s: Here you go Sir, take this food!",
+  "com.minecolonies.coremod.pirates.sailing.away": "A pirate ship %s of %s just sailed away to search for another victim.",
+  "com.minecolonies.coremod.cook.serve.player": "%s: Here you go mayor, take this food!",
   "block.minecolonies.blockhutarchery": "Archery",
   "block.minecolonies.blockhutarchery.name": "Archery",
   "com.minecolonies.coremod.gui.workerhuts.archery": "Archery",
   "block.minecolonies.blockhutcombatacademy": "Combat Academy",
   "block.minecolonies.blockhutcombatacademy.name": "Combat Academy",
 
   "com.minecolonies.coremod.gui.workerhuts.combatacademy": "Combat Academy",
-  "item.minecolonies.santa_hat": "Santa hat",
+  "item.minecolonies.santa_hat": "Santa Hat",
 
   "com.minecolonies.coremod.gui.workerhuts.sawmill": "Sawmill",
   "block.minecolonies.blockhutsawmill": "Sawmill",
   "block.minecolonies.blockhutsawmill.name": "Sawmill",
 
-  "com.minecolonies.coremod.recipe.unable": "Unable to store the blueprint. Either the %s has no space for any more blueprints, or the blueprint doesn't correspond with its profession.",
+  "com.minecolonies.coremod.recipe.unable": "Unable to store the recipe. Either the %s has no space for any more recipes, or the recipe doesn't correspond with its profession.",
   "com.minecolonies.coremod.request.crafting": "Crafter",
-  "entity.builder.messagebuilderstoofar": "There is no builder close enough to work at this building, assign a builder to it manually in the build screen or build it closer to the existing builders!",
-  "com.minecolonies.coremod.bakery.norecipe": "Please tell me which recipe you want me to prepare in the Bakery!",
+  "entity.builder.messagebuilderstoofar": "There is no Builder close enough to work on this building! Assign a Builder to it manually in the build screen or build it closer to the existing Builders.",
+  "com.minecolonies.coremod.bakery.norecipe": "%s: Please tell me which recipe you want me to bake!",
```

Comment:
This is another one I believe is chitchat bubbles, and not in chat, so is likely redundant.